client.go

```diff
@@ -7,15 +7,14 @@ import (
 	"net/http"
 	"net/url"
 	"strings"
-
-	"github.com/Sirupsen/logrus"
```

Comment:
Why remove the logs - These are debug only, which can be turned off based on a flag?
These can be helpful when debugging requests.